java/code/src/com/suse/manager/hub/DefaultHubInternalClient.java

```diff
@@ -66,26 +67,30 @@ public DefaultHubInternalClient(String remoteHostIn, String tokenIn, Optional<Ce
 
     @Override
     public void registerHub(String token, String rootCA) throws IOException {
-        invokePostMethod("registerHub", new RegisterJson(token, rootCA), Void.class);
+        invokePostMethod(List.of("hub", "sync"), "registerHub", new RegisterJson(token, rootCA), Void.class);
```

Comment:
Do you see any benefits in using a list here? I think we can just pass the namespace as "hub/sync".

Reply:
This prevent errors in defining the namespace. But if you think we can also use just a string.

Reply:
changed back to string